src/components/UniversalNavbarExpanded/UniversalNavbarExpanded.js

```diff
@@ -0,0 +1,408 @@
+          key={link.id}
+          href={link.href}
+          LinkComponent={link.href !== '/login/' ? LinkComponent : null}
+        >
+          <SearchIcon />
+        </NavLink>
+      )
+    }
+
+    const renderAccountIcon = (link) => {
+      return (
+        <NavLink
+          className={styles.accountIcon}
+          key={link.id}
+          href={link.href}
+          LinkComponent={link.href !== '/login/' ? LinkComponent : null}
+        >
+          <AccountIcon />
+        </NavLink>
+      )
+    }
+
+    const Hamburger = () => (
+      <div className={styles.hamburger}>
+        <TransformingBurgerButton
+          showMobileMenu={showMobileMenu}
+          clickHandler={this.toggleHamburger}
+          keyPressHandler={this.handleHamburgerKeyPress}
+        />
+      </div>
+    )
+
+    return (
+      <div className={styles.blockNavbar}>
+        <div className={styles.navbar}>
+          <Layout.ScrollDetector>
+            <Hamburger />
+            <div
+              className={
+                showMobileMenu
+                  ? [styles.phoneAndTablet, styles.visible].join(' ')
+                  : styles.phoneAndTablet
+              }
+            >
+              <div
+                className={
+                  showMobileMenu ? styles.mobileMenu : styles.hideMobileMenu
+                }
+              >
+                <NavLink
+                  href={logoHref}
+                  LinkComponent={LinkComponent}
+                  className={styles.phoneLogo}
+                >
+                  {LogoWhite({ className: styles.logo })}
+                </NavLink>
+                <Hamburger />
+                <div className={styles.accordion}>
+                  {LINKS.NAVLINKS.map((link, idx) => (
+                    <div
+                      key={link.id}
+                      className={
+                        idx === activeAccordionItem
+                          ? [styles.accordionItem, styles.active].join(' ')
+                          : styles.accordionItem
+                      }
+                      onClick={() => this.toggleAccordionItem(idx)}
+                      onKeyPress={(e) =>
+                        this.handleAccordionItemKeyPress(e, idx)
+                      }
+                      tabIndex={0}
+                      role="button"
+                    >
+                      <div className={styles.accordionParent}>
+                        <TitleMedium.Sans.Regular400>
+                          {link.title}
+                        </TitleMedium.Sans.Regular400>
+                        <AccordionToggleIcon />
+                      </div>
+                      <div className={styles.accordionChildren}>
+                        <div
+                          key={link.subnav.cta.id}
+                          className={styles.accordionChild}
+                        >
+                          <TitleSmall.Sans.Light300>
+                            <NavLink
```

Comment:
I just reviewed similar code from victor. There’s a good deal of duplication in these nav links